src/AddressRegistry.Api.Oslo/AddressMatch/Extensions.cs

```diff
@@ -42,6 +42,9 @@ public static string ToggleAbbreviations(this string input)
             if (input.Contains("o.l.v."))
                 return input.Replace("o.l.v.", "Onze Lieve Vrouw");
 
+            if (input.Contains("onze-lieve-"))
```

Comment:
Can we add another InlineData here? Or do they cover a different scenario?
https://github.com/Informatievlaanderen/address-registry/blob/main/test/AddressRegistry.Tests/ApiTests/AddressMatch/AddressMatchV2Tests.cs#L275